package.json

```diff
@@ -1,45 +1,45 @@
 {
-  "name": "webui",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
-  "scripts": {
-    "dev": "npm run dev:liveion",
-    "dev:liveion": "vite -c web/liveion/vite.config.ts",
-    "dev:liveman": "vite -c web/liveman/vite.config.ts",
-    "preview": "npm run preview:liveion",
-    "preview:liveion": "vite preview -c web/liveion/vite.config.ts",
-    "preview:liveman": "vite preview -c web/liveman/vite.config.ts",
-    "lint": "eslint && tsc --noEmit",
-    "build": "npm run lint && npm run build:liveion && npm run build:liveman",
```

Comment:
Please do not remove the lint. We should run `npm run lint` locally and fix all the errors reported by eslint.